### ISSUE TITLE
require is not defined

### ISSUE THREAD

**Issue by mikob** [ChrisOh431/astro-remark-eleventy-image#10]
Thanks for this plugin! I just tried installing it but right away getting an error :/

```
> @example/basics@0.0.1 start /home/mikob/workspace/angulo-cowork
> astro dev

6:39:43 PM [vite] Error when evaluating SSR module /node_modules/.pnpm/@11ty+eleventy-img@3.0.0/node_modules/@11ty/eleventy-img/img.js:

6:39:43 PM [vite] Error when evaluating SSR module /node_modules/.pnpm/astro-remark-eleventy-image@2.0.1/node_modules/astro-remark-eleventy-image/src/remark-plugin.ts:

6:39:43 PM [vite] Error when evaluating SSR module /node_modules/.pnpm/astro-remark-eleventy-image@2.0.1/node_modules/astro-remark-eleventy-image/index.ts:

6:39:43 PM [vite] Error when evaluating SSR module /home/mikob/workspace/angulo-cowork/astro.config.mjs:

06:39:43 PM [astro] Unable to load /home/mikob/workspace/angulo-cowork/astro.config.mjs

 error   require is not defined
  File:
    /node_modules/.pnpm/@11ty+eleventy-img@3.0.0/node_modules/@11ty/eleventy-img/img.js:3:27
  Stacktrace:
ReferenceError: require is not defined
    at eval (/node_modules/.pnpm/@11ty+eleventy-img@3.0.0/node_modules/@11ty/eleventy-img/img.js:3:27)
    at instantiateModule (file:///home/mikob/workspace/angulo-cowork/node_modules/.pnpm/vite@4.2.1/node_modules/vite/dist/node/chunks/dep-79892de8.js:53996:15)
```

My config is pretty standard, and I don't have SSR enabled, using typescript...
```
import path from "path";
import { fileURLToPath } from "url";
import { defineConfig } from "astro/config";
import tailwind from "@astrojs/tailwind";
import sitemap from "@astrojs/sitemap";
import image from "@astrojs/image";
import mdx from "@astrojs/mdx";
import partytown from "@astrojs/partytown";
import compress from "astro-compress";
import { SITE } from "./src/config.mjs";
import remarkEleventyImage from "astro-remark-eleventy-image";
const __dirname = path.dirname(fileURLToPath(import.meta.url));
const whenExternalScripts = (items = []) => SITE.googleAnalyticsId ? Array.isArray(items) ? items.map(item => item()) : [items()] : [];


// https://astro.build/config
export default defineConfig({
  site: SITE.origin,
  base: SITE.basePathname,
  trailingSlash: SITE.trailingSlash ? "always" : "never",
  output: "static",
  integrations: [tailwind({
    config: {
      applyBaseStyles: false
    }
  }), sitemap(), image({
    serviceEntryPoint: "@astrojs/image/sharp"
  }), mdx(), ...whenExternalScripts(() => partytown({
    config: {
      forward: ["dataLayer.push"]
    }
  })), compress({
    css: true,
    html: {
      removeAttributeQuotes: false
    },
    img: false,
    js: true,
    svg: false,
    logger: 1
  }), remarkEleventyImage()],
  markdown: {},
  vite: {
    resolve: {
      alias: {
        "~": path.resolve(__dirname, "./src")
      }
    }
  }
});
```

**Comment by seesjays** [comment]
Thanks so much for pointing this out, it's been a while since I've had to mess with this. I looked into it and it seems that it's an issue with Astro's Vite version. A [similar package](https://www.npmjs.com/package/astro-eleventy-img) (that inspired this one) also ran into this issue, and fixed it by advising in the Installation section to add
```js
vite: {
    ssr: {
      external: ["@11ty/eleventy-img"],
    },
},
```
to the **astro-config** file.

I wish there were a simpler fix for this that I could do to the package itself, but for now things are working on my end with this tweak.
